packages/loopring_v3/contracts/impl/libexchange/ExchangeBlocks.sol

```diff
@@ -1,33 +1,26 @@
-/*
-
-  Copyright 2017 Loopring Project Ltd (Loopring Foundation).
-
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-  http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
-*/
-pragma solidity ^0.6.6;
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.6.10;
 pragma experimental ABIEncoderV2;
 
 import "../../lib/AddressUtil.sol";
-import "../../lib/BytesUtil.sol";
 import "../../lib/MathUint.sol";
 
+import "../../thirdparty/BytesUtil.sol";
+
 import "../../iface/IBlockVerifier.sol";
-import "../../iface/IDecompressor.sol";
 import "../../iface/ExchangeData.sol";
 
 import "./ExchangeMode.sol";
 import "./ExchangeWithdrawals.sol";
 
+import "../libtransactions/TransferTransaction.sol";
+import "../libtransactions/AccountUpdateTransaction.sol";
+import "../libtransactions/DepositTransaction.sol";
+import "../libtransactions/WithdrawTransaction.sol";
+import "../libtransactions/OwnerChangeTransaction.sol";
```

Comment:
@Brecht, is it right to say that besides Forced Withdrawals, there is no other types of requests that the relayer has to process? I would like the support for Stateless Wallet to be added to the relayer in a later stage.

Reply:
That's right.